src/lib.rs

```diff
@@ -208,4 +359,44 @@ mod tests {
         assert_eq!(2, refund_transaction.output[1].value);
         assert_eq!(3, refund_transaction.input[0].sequence);
     }
+    #[test]
+    fn create_cet_transaction_test() {
+        let addr = Address::from_str("33iFwdLuRpW1uK1RTRqsoi8rR4NpDzk66k").unwrap();
+
+        let maturity_time = 500000001;
+        let cet_script = vec![0u8, 1, 2, 3];
```

Comment:
It would be nice to also test the actual cet script creation.

Reply:
I agree.  I'll add a test for this.